src/riscv-integration.adoc

```diff
@@ -1078,27 +1087,79 @@ memory addresses must match for the address to be valid:
 * For Sv48, bits [63:48] must equal bit 47
 * For Sv57, bits [63:57] must equal bit 56
 
-RISC-V permits that some CSRs, such as <<mtvec>> and <<mepc>> (see
-xref:CSR_exevectors[xrefstyle=short]), need not be able to hold all possible
-invalid addresses. Prior to writing these CSRs, implementations may convert an
-invalid address into some other invalid address that the register is capable of
-holding. However, these registers hold capabilities in {cheri_base_ext_name}
-and the bounds encoding depends on the address value, so implementations must
-not convert invalid addresses to other arbitrary invalid addresses in an
-unrestricted manner.
-The following procedure must be used instead when writing a capability A to
-these CSRs:
-
-. If A's address cannot be held then convert it to another address that the CSR can
-hold
-. If conversion _was_ required, then A's tag is cleared if A is
-sealed or if the new address is not representable -- this is equivalent to the
-semantics of <<SCADDR>>
+RISC-V permits that CSRs holding addresses, such as <<mtvec>> and <<mepc>>
+(see xref:CSR_exevectors[xrefstyle=short]) as well as <<pcc>>, need not
```

Comment:
Should this refer to pc rather than pcc?

Reply:
Ah yes, you are right. I will fix it!

Reply:
Fixed!